src/nl/hannahsten/texifyidea/inspections/latex/LatexLabelConventionInspection.kt

```diff
@@ -29,6 +29,10 @@ open class LatexLabelConventionInspection : TexifyInspectionBase() {
         private fun getLabeledCommand(label: PsiElement): PsiElement? {
             return when (label) {
                 is LatexCommands -> {
+                    if (Magic.Command.labelAsParameter.contains(label.commandToken.text)) {
```

Comment:
Why use `command.commandToken.text` everywhere instead of `command.name`?

Reply:
I totally missed that `command.name` means the same, thanks for pointing it out.

Reply:
Actually it's not the same: `command.name` is from the index, while `command.commandToken.text` is not, I think. I'm not aware of any performance improvement but it just makes sense to me to use the indexed name.